src/main/java/br/codenation/hospital/manager/model/Hospital.java

```diff
@@ -1,19 +1,25 @@
 package br.codenation.hospital.manager.model;
 
-import lombok.AllArgsConstructor;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.ToString;
+import lombok.*;
+import org.springframework.data.annotation.Id;
+
+import java.util.List;
 
 @Getter
+@Setter
 @ToString
 @EqualsAndHashCode
 @AllArgsConstructor
```

Comment:
https://projectlombok.org/features/Data. Check this out! Therefore you only need @Data and @AllArgsConstructor

Reply:
This apply to all models